core/src/main/scala/org/apache/spark/FutureAction.scala

```diff
@@ -58,7 +58,7 @@ trait FutureAction[T] extends Future[T] {
    *
    * @param atMost maximum wait time, which may be negative (no waiting is done), Duration.Inf
    *               for unbounded waiting, or a finite positive duration
-   * @throws Exception exception during action execution
+   * @note Throws `Exception` exception during action execution
```

Comment:
I see. Yeah it's unfortunate that this (and other) classes aren't found by unidoc. They're JDK classes! I don't see what we could be doing wrong though,

In other cases it's easy enough to change `[[` to a back-tick to work around the warning. Here you'd have to mis-use scaladoc a bit to work around it. I think you could leave this, IMHO. The other `[[` changes make sense.